docs/rules/object-property-newline.md

```diff
@@ -72,20 +71,53 @@ Another benefit of this rule is specificity of diffs when a property is changed:
 +var obj = { foo: "foo", bar: "bazz", baz: "baz" };
 ```
 
-### Optional Exception
+### Optional Exceptions
+
+The rule offers three object option.
 
-The rule offers one object option, `allowMultiplePropertiesPerLine`. If you set it to `true`, object literals such as the first two above, with all property specifications on the same line, will be permitted, but one like
+#### `allowMultiplePropertiesPerLine`
+
+If you set `allowMultiplePropertiesPerLine` to `true`, object literals such as the first two above, with all property specifications on the same line, will be permitted, but one like
 
 ```js
 const newObject = {
     a: 'a.m.', b: 'p.m.',
     c: 'daylight saving time'
 };
-
 ```
 
 will be prohibited, because two properties, but not all properties, appear on the same line.
 
+#### `treatComputedPropertiesLikeJSCS`
```

Comment:
I think this option name should be more generic and not reference JSCS directly (although the option *description* can certainly reference JSCS). Maybe something like `ignoreBracketsInComputedProperties`?